package.json

```diff
@@ -74,7 +74,7 @@
     "prettier-plugin-svelte": "^2.9.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "storybook": "^7.0.0-beta.13",
+    "storybook": "^7.0.0-rc.8",
```

Comment:
Do the various `@storybook/*` packages require a bump, too?

Reply:
Yes. Thx.

Reply:
Actually I'm not sure if we need to update this since it's just for docs?

Reply:
I went ahead and bumped.

Reply:
Nice, thanks for doin' this!